lib/rspec/support/ruby_features.rb

```diff
@@ -55,6 +55,18 @@ def caller_locations_supported?
         respond_to?(:caller_locations, true)
       end
 
+      begin
```

Comment:
`Exception.method_defined?(:cause)`

No need for a linear `include?` operation.